### ISSUE TITLE
ghc-head shouldn't be the default haskell compiler

### ISSUE THREAD

**Issue by Mee42** [Headline/discord-compiler-bot#15]
At the current moment, running ;compile haskell will result in a compiler error `ghc: failed to create OS thread: Cannot allocate memory`

**Comment by Headline** [comment]
We don’t control what the default compiler is, that is definitely a WandBox issue but we can have a patch on our end that prevents fatal errors like this from occurring, any recommendations what the default for Haskell should be?

**Comment by Mee42** [comment]
Here's some more testing:


;compile ghc-head
```
main = putStrLn "hi"
```
Link: https://wandbox.org/permlink/g5baIjZ7XRUSkoHv
Output: ghc: failed to create OS thread: Cannot allocate memory


;compile haskell
```
main = putStrLn "hi"
```
Link: https://wandbox.org/permlink/g5hcBxUjNm1pamej
Output: ghc: failed to create OS thread: Cannot allocate memory


;compile ghc-8.4.2
```
main = putStrLn "hi"
```
Link: https://wandbox.org/permlink/6d7FXFysSzpZ7AV6
Output: hi

**Comment by Mee42** [comment]
The default haskell compiler should be the latest release of ghc available on wandbox, which is at the moment is `ghc-8.4.2`

**Comment by Headline** [comment]
Fixed in #17, thanks!
